### ISSUE TITLE
[example-request] btrfs and bcachefs raids without lvm/mdam

### ISSUE THREAD

**Issue by con-f-use** [nix-community/disko#543]
It's a bit hard for newbies such as myself to grog the details of disko config just from the examples without a detailed documentation. Does disko actually support btrfs- and/or bachefs-raids without using mdam? If yes, could we have examples for that. I trust there will be formal reference documentation at one point, once you figured out to generate it from the module types, right?

**Comment by dailyherold** [comment]
I'd be interested as well in this. I ended up going single disk then the `btrfs` commands post-install to add and convert to raid1. Searching "language:nix disko btrfs raid" and I did find [this](https://github.com/tiredofit/nixos-config/blob/da4b765d8d496e821828d173705fcef010dae10f/deploy-templates/disko/efi-btrfs-swap-raid1.nix#L39). Might try out but syntax didn't seem native btrfs which I was trying to maintain.
